pkgs/tools/security/feroxbuster/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, perl
+, rustPlatform
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "feroxbuster";
+  version = "2.6.1";
+
+  src = fetchFromGitHub {
+    owner = "epi052";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-RY9bFuALRaVXDrC0eIx0inPjRqNpRKNZf3mCrKIdGL8=";
+  };
+
+  cargoSha256 = "sha256-0Zawlx/lhF7K8nOsHYKO84pnctVMpm3RfnAFCOltOqE=";
+
+  nativeBuildInputs = [
+    perl
```

Comment:
Likely this is only needed for the vendored openssl. 

Adding `pkgs-config`, `openssl` as inputs and setting `OPENSSL_NO_VENDOR` should work.

Reply:
Thanks for the feedback, @zowoq.